npc/re/merchants/cashmall.txt

```diff
@@ -329,3 +330,54 @@ itemmall,19,74,5	script	Stylist#cash	91,{
 	mes "to see you again. Take care!";
 	close;
 }
+
+itemmall,41,55,3	script	Cat Hand Salesman Tunkarom#	4_M_MERCAT1,{
+	mes "[Tunkarom]";
+	mes "Welcome!";
+	mes "If you have a special coupon, we'll exchange it for the product you want!";
+	mes "These are exchangeable coupons.";
+	mes "^i[1001583] ^i[1001581] ^i[1001584] ^i[1001588] ^i[1001809] ^i[1001810]";
```

Comment:
Should we hide those behind a script command (for checking supported clients) too?

Reply:
yes, we could do it